container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -706,89 +709,12 @@ impl Contains<RuntimeCall> for NormalFilter {
     }
 }
 
-pub struct NormalDmpHandler;
-impl DmpMessageHandler for NormalDmpHandler {
-    // This implementation makes messages be queued
-    // Since the limit is 0, messages are queued for next iteration
-    fn handle_dmp_messages(
-        iter: impl Iterator<Item = (RelayBlockNumber, Vec<u8>)>,
-        limit: Weight,
-    ) -> Weight {
-        (if Migrations::should_pause_xcm() {
-            DmpQueue::handle_dmp_messages(iter, Weight::zero())
-        } else {
-            DmpQueue::handle_dmp_messages(iter, limit)
-        }) + <Runtime as frame_system::Config>::DbWeight::get().reads(1)
-    }
-}
-
-pub struct MaintenanceDmpHandler;
-impl DmpMessageHandler for MaintenanceDmpHandler {
-    // This implementation makes messages be queued
-    // Since the limit is 0, messages are queued for next iteration
-    fn handle_dmp_messages(
-        iter: impl Iterator<Item = (RelayBlockNumber, Vec<u8>)>,
-        _limit: Weight,
-    ) -> Weight {
-        DmpQueue::handle_dmp_messages(iter, Weight::zero())
-    }
-}
-
-/// The hooks we want to run in Maintenance Mode
-pub struct MaintenanceHooks;
```

Comment:
Ah I see, so we only needed to override this because of the on_idle right? excluding on_idle (which we are changing) all others are `AllPalletsWithSystem`?

Reply:
That's it!